app/controllers/photoTagController.js

```
@@ -42,13 +41,39 @@ export default class PhotoTagController {
   }
 
   /**
+   * Deletes a PhotoTag given the tagName within the url param
    *
    * @param {Request} req
    * @param {Response} res
    */
-  static async getTagByName(req, res) {
+  static async delete(req, res) {
     try {
+      const tagName = req.params.tagName;
+      const tag = await PhotoTagAccessor.getTagByName(tagName);
+
+      if (!tag) {
+        throw new ErrorPhotoTagNotFound();
+      }
```

Arushi's argument is that this call to the database to get all the data about the `photoTag` is unecessary, since you can delete by the tagName itself. 

The next question would be, how can I then tell if the tag actually existed? Since you'd want to throw an error if the you didn't end up deleting something.